osu.Game/Overlays/Dashboard/CurrentlyPlayingDisplay.cs

```diff
@@ -35,35 +37,42 @@ internal class CurrentlyPlayingDisplay : CompositeDrawable
 
         private SearchContainer<PlayingUserPanel> userFlow;
 
-        private FocusedTextBox searchBar;
-        private Container<FocusedTextBox> searchBarContainer;
+        private Box searchBarBackground;
+        private BasicSearchTextBox searchBar;
+        private Container<BasicSearchTextBox> searchBarContainer;
 
         [Resolved]
         private SpectatorClient spectatorClient { get; set; }
 
         [BackgroundDependencyLoader]
-        private void load()
+        private void load(OverlayColourProvider colourProvider)
         {
             RelativeSizeAxes = Axes.X;
             AutoSizeAxes = Axes.Y;
 
-            searchBarContainer = new Container<FocusedTextBox>
+            searchBarBackground = new Box
+            {
+                RelativeSizeAxes = Axes.X,
+                Height = 10*2 + search_bar_height,
+                Colour = colourProvider.Background4,
+            };
+
+            searchBarContainer = new Container<BasicSearchTextBox>
             {
                 Anchor = Anchor.TopCentre,
                 Origin = Anchor.TopCentre,
+                RelativeSizeAxes = Axes.X,
                 Padding = new MarginPadding(10),
-                Child = searchBar = new FocusedTextBox
+
+                Child = searchBar = new BasicSearchTextBox
                 {
                     Anchor = Anchor.TopCentre,
                     Origin = Anchor.TopCentre,
                     Height = search_bar_height,
-                    Width = search_bar_width,
 
-                    Colour = OsuColour.Gray(0.8f),
+                    RelativeSizeAxes = Axes.X,
 
-                    PlaceholderText = "Search for User...",
-                    HoldFocus = true,
-                    ReleaseFocusOnCommit = true,
+                    PlaceholderText = "type to search",
```

Comment:
Please look at other usages of this. It's localisable.

Reply:
done!